WTWSMS/events/VIET_Jewish_Flavor_Events.txt

```diff
@@ -3,7 +3,7 @@
 
 #FLAVOR EVENTS
 
-#A bigoted courtier questions why we must have Jews in the court
+#A courtier questions why we must have Jews in the court
```

Comment:
You are absolutely wrong. There is no reason to have soapbox commentary in the code. You rail against me when I do something like this. That crap goes.